content/docs/command-reference/fetch.md

```diff
@@ -179,8 +179,8 @@ Note that the `.dvc/cache` directory was created and populated.
 
 Used without arguments (as above), `dvc fetch` downloads all files and
 directories needed by all `dvc.yaml` and `.dvc` files in the current branch. For
-example, the hash values `3863d0e...` and `42c7025...` correspond to the
-`model.pkl` file and `data/features/` directory, respectively.
+example, the hash value `20b786b...` corresponds to the `data/prepared/`
+directory.
```

Comment:
Both hash values should be kept (for the model file and for the data/prep dir)

Reply:
Hash value for `model.pkl` is `c8d30..`. Since it wasn't visible in `tree` displayed above so I removed it. Have updated now.